package.json

```diff
@@ -25,31 +25,30 @@
 		"url": "https://github.com/gisat/ptr-locales"
 	},
 	"dependencies": {
-		"i18next": "^19.8.7",
-		"lodash": "^4.17.20",
+		"i18next": "^20.3.2",
+		"lodash": "^4.17.21",
 		"react": "16.13.1",
-		"react-i18next": "9.0.10"
+		"react-i18next": "^11.11.0"
```

Comment:
update react to ^16.13.1

Reply:
Yep.
Fixed: 5f301e990879ef53b30da10804b4187d05edc415